### DISCUSSION TITLE
Better `CONTRIBUTING.md`

### DISCUSSION THREAD

**Discussion by LimpidCrypto** [loco-rs/loco#297] [Ideas]
We should improve the `CONTRIBUTING.md`. As you have stated [here](https://github.com/loco-rs/loco/discussions/108#discussioncomment-7797304) you are seeking for more code contributions. I was about to add a blog post about how  to set up loco with an Angular frontend. Taking a look at the `CONTRIBUTING.md` I don't get any clues where to start. I don't know what tools to use, if there are possiblities to host the blog locally, etc.?
We should have general contributing guidelines like:
- Fork the repo
- Clone the fork
- Create a new branch
- ...
- Testing
- Linters

And we should have guidelines more specific about what I want to contribute.
- What do I need to create a new blog post?
- What do I need to contribute code?

I'm curious about what you're thinking about it and your ideas what else to include.

**Comment by LimpidCrypto** [comment]
@kaplanelad maybe you can help me with contributing my blog post? It seems like you are using a markdown tool like Hugo or similar to generate new posts?

**Reply by kaplanelad** [reply]
@LimpidCrypto I'll push better documentation into the contributing.md. 
In the meantime, to run the docs website, we are using [zola](https://www.getzola.org/). 
The steps that you should take:
1. download the zola binary
2. cd to `docs-site`
3. run `zola serve`

**Reply by LimpidCrypto** [reply]
Awesome, thank you 👍🏼
